src/screens/Profile/Home/index.tsx

```diff
@@ -175,17 +191,48 @@ export default function ProfileHome({navigation}: ProfileScreenProps) {
             />
           </Sections.Section>
         )}
+
+        <View style={style.debugInfoContainer}>
+          {config?.installId && (
+            <View>
+              {fadeIsAnimating && (
+                <ClipboardFadeContainer>
+                  <ScreenReaderAnnouncement
+                    message={t(ProfileTexts.installId.wasCopiedAlert)}
+                  />
+                  <ThemeText>
```

Comment:
Copy theme text here is wrong color but I think this should be solved with merging master in